### ISSUE TITLE
Track position slider behaving weirdly

### ISSUE THREAD

**Issue by SourcedX777** [SpotCompiled/SpotC-Plus-Plus#48]
For some users, the track position slider in Spotify may act weird.
By weird, I mean you have to use the track position slider from the lock screen, because the one from within the now playing view in Spotify may seem to be stuck.

**Comment by yodaluca23** [comment]
> It’s an accessibility issue with VoiceOver so I don't know how to explain it without a screen recording. I'll make one later today. Sent from Xavier FucciHave a nice day!On Oct 10, 2024, at 6:58 AM, Luca LeBlanc ***@***.***> wrote:﻿
> 
> Can you send a screen recording, I am unable to reproduce the described behavior.
> 
> 
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

so then does it happen in the vanilla Spotify app?

**Comment by SourcedX777** [comment]
The version that I'm using now, this accessibility problem exists when the overwrite configuration setting is off.Turning it on fixes the issuevery in v2.2.8 but in any newer version, it no longer fixes the issue so that's the weird myster about this. Sent from Xavier FucciHave a nice day!On Oct 10, 2024, at 9:44 AM, Luca LeBlanc ***@***.***> wrote:﻿

It’s an accessibility issue with VoiceOver so I don't know how to explain it without a screen recording. I'll make one later today. Sent from Xavier FucciHave a nice day!On Oct 10, 2024, at 6:58 AM, Luca LeBlanc @.***> wrote:﻿
Can you send a screen recording, I am unable to reproduce the described behavior.
—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.***>

so then does it happen in the vanilla Spotify app?

**Comment by yodaluca23** [comment]
> The version that I'm using now, this accessibility problem exists when the overwrite configuration setting is off.Turning it on fixes the issuevery in v2.2.8 but in any newer version, it no longer fixes the issue so that's the weird myster about this.

In EeveeSpotify swift5.3 https://github.com/whoeevee/EeveeSpotify/releases/tag/swift5.3 (2.2.9) the override configuration was updated, probably has to do with this...

**Comment by yodaluca23** [comment]
@SourcedX777 Is this still happening?

**Comment by SourcedX777** [comment]
Yes, It still happens.I can try again the next time there is a new release if you like, but nothing's changed since a long time ago, and that was the thing that broke accessibility of the track position control in the first place.Sent from my iPhoneOn Apr 12, 2025, at 5:45 PM, Luca LeBlanc ***@***.***> wrote:﻿
  @SourcedX777 Is this still happening on latest?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>



yodaluca23 left a comment (SpotCompiled/SpotC-Plus-Plus#48)
@SourcedX777 Is this still happening on latest?
